zk-sdk/src/encryption/pod/auth_encryption.rs

```diff
@@ -5,21 +5,29 @@ use crate::{encryption::auth_encryption::AeCiphertext, errors::AuthenticatedEncr
 use {
     crate::{
         encryption::AE_CIPHERTEXT_LEN,
-        pod::{impl_from_bytes, impl_from_str},
+        pod::{impl_from_bytes, impl_from_str, impl_wasm_bindings},
     },
     base64::{prelude::BASE64_STANDARD, Engine},
     bytemuck::{Pod, Zeroable},
     std::fmt,
 };
+#[cfg(target_arch = "wasm32")]
+use {
+    js_sys::{Array, Uint8Array},
```

Comment:
nit: it's typically easier for the macro to use fully qualified names or import its own required types, rather than force callers of `impl_wasm_bindings` to import what the macro needs